### PR TITLE
Add content for Community page

### PR DESCRIPTION
**Pull request by chenette**
Contains a brief "contributing code" section

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+Splinter is an open source project on GitHub. We encourage you to join the
+Splinter community and contribute to this project in the following ways:
+
+**Chat** at [splinterworld.slack.com](http://splinterworld.slack.com).
+Join us on Slack to ask questions, tell us about a bug, or propose changes and
+new features.
+
+**Report bugs**: We track bugs as GitHub issues at
+[Cargill/splinter/issues](https://github.com/Cargill/splinter/issues).
+GitHub requires a Linux Foundation ID; you can register for a free account at
```
Comment (vaporos) [inline community/index.md:14]: LF id isn't relevant here.

Comment (chenette) [reply community/index.md]: Removed the LFID sentence.

File: community/index.md [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+Welcome!
+
+Splinter is an open source project on GitHub. We encourage you to join the
+Splinter community and contribute to this project in the following ways:
+
+**Chat** at [splinterworld.slack.com](http://splinterworld.slack.com).
+Join us on Slack to ask questions, tell us about a bug, or propose changes and
+new features.
+
+**Report bugs**: We track bugs as GitHub issues at
```
Comment (davececchi) [inline community/index.md:12]:
```suggestion
**Bi-Weekly Engineering Forums**: We host an [open zoom meeting](https://us02web.zoom.us/j/89560362141) to provide brief updates on Splinter development and to provide a way for the community to connect with each other and with us.   Every other Thursday from 8:00 - 9:00 US-Central time.

**Report bugs**: We track bugs as GitHub issues at
```

Comment (chenette) [reply community/index.md:12]: Great idea!  I'll do that in a separate PR so that this one can be merged asap.

**Pull request merged**